src/api/api_models.rs

```diff
@@ -186,7 +186,7 @@ trait WithImages {
 pub struct Playlist {
     pub id: String,
     pub name: String,
-    pub images: Vec<Image>,
+    pub images: Option<Vec<Image>>,
```

Comment:
Does `#[serde(default)]` fix the issue?

Reply:
Surprisingly, no. I haven't investigated why but adding that to the images will still throw a null-errror.